cabal.project

```diff
@@ -110,49 +110,49 @@ source-repository-package
   type: git
   location: https://github.com/input-output-hk/iohk-monitoring-framework
   subdir: iohk-monitoring
-  tag: c35f302ad39568b9813f20a0108049627bd7ace0
+  tag: e0eba52e8e7c81d3f002c8d36a3e5f6ed30bb216
```

Comment:
It seems like you accidentally changed the hashes of `iohk-monitoring-framework` instead of `ouroboros-network` in this file.

Reply:
Yep, saw about the Nix deps, missed the wrong project. FIxed.